### ISSUE TITLE
Add L-alpha (and L-beta?) fitters for a few platinum-group elements

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#210]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/210) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

We’re using Pt L-alpha lines for tomography with Tomcat. We should add at least a `PtLAlpha` model, but I suggest L-alpha and L-beta1 for at least Ir, Pt, and Au. 

Here’s the code that we are not quite using for Pt L-alpha, but we could…

```python
mass.addline(
    element="Pt",
    linetype="LAlpha",
    material="Pt",
    reference_short='NIST ASD',
    nominal_peak_energy=mass.STANDARD_FEATURES['PtLAlpha1'],
    energies=np.array([mass.STANDARD_FEATURES['PtLAlpha2'],
                       mass.STANDARD_FEATURES['PtLAlpha1']]),
    lorentzian_fwhm=np.array([7.5, 7.5]),  # was 5.53, 9.8 before
    reference_amplitude=np.array([1.0, 9.0]),
    reference_amplitude_type=mass.calibration.LORENTZIAN_PEAK_HEIGHT,
    ka12_energy_diff=None,
    fitter_type=mass.line_fits.GenericKAlphaFitter,
    reference_measurement_type="Experiment"
)
```

Probably it’s a good idea to get the widths from Campbell & Papp 2003 and the energies from Deslattes et al 2003.

@{557058:9524f6a0-ef9f-4466-b913-d04e51ff9093} Is there a reason we didn’t try this? Is there a reason for the comment that widths of 7.5 and 7.5 “used to be 5.53, 9.8” eV?

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Add some platinum-group L lines

Add LAlpha and LBeta1 lines of (Ir, Pt, Au, Pb, Bi) from Zschornack. Fixes #210.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)
